src/main/java/com/sarahhand/fractals/model/MandelbrotPointData.java

```diff
@@ -0,0 +1,39 @@
+package com.sarahhand.fractals.model;
+
+import java.awt.geom.Point2D.Double;
+
+/**
+ * Represents the data needed for a color scheme to color a point in the Mandelbrot set
+ * @author J9465812
+ */
+public class MandelbrotPointData implements PointData{
+	
+	private Double location;
+	private int escapeTime;
+	private Double z;
+	
+	public MandelbrotPointData(Double location, int escapeTime, Double z){
+		this.location = location;
+		this.escapeTime = escapeTime;
+		this.z = z;
+	}
+	
+	public Double getLocation(){
+		return location;
+	}
+	public void setLocation(Double location){
```

Comment:
Does this class need to have any set methods?  If there isn't a strong reason to keep them, I'd get rid of them.